src/portal/src/app/project/summary/summary.component.html

```diff
@@ -0,0 +1,72 @@
+        <li>{{ summaryInformation.project_admin_count }} {{'SUMMARY.ADMIN' | translate}}</li>
+        <li>{{ summaryInformation.master_count }} {{'SUMMARY.MASTER' | translate}}</li>
+        <li>{{ summaryInformation.developer_count }} {{'SUMMARY.DEVELOPER' | translate}}</li>
+        <li>{{ summaryInformation.guest_count }} {{'SUMMARY.GUEST' | translate}}</li>
+      </ul>
+    </div>
+  </div>
+  <div class="summary-right pt-1">
+    <div class="display-flex  project-detail">
+      <h5 class="mt-0">{{'SUMMARY.PROJECT_QUOTAS' | translate}}</h5>
+      <div class="ml-1">
+        <div class="display-flex quotas-progress">
+          <label class="mr-1">{{'SUMMARY.ARTIFACT_COUNT' | translate}}</label>
+          <label class="progress-label">{{ summaryInformation.quota.used.count }} {{ 'QUOTA.OF' | translate }}
+            {{ summaryInformation.quota.hard.count ===-1?('QUOTA.UNLIMITED' | translate): summaryInformation.quota.hard.count }}
+          </label>
+        </div>
+
+        <div>
+          <div class="progress-block progress-min-width progress-div">
+            <div class="progress success" [class.danger]="summaryInformation.quota.hard.count!==-1?summaryInformation.quota.used.count/summaryInformation.quota.hard.count>0.9:false">
+              <progress
+                value="{{summaryInformation.quota.hard.count===-1? 0 : summaryInformation.quota.used.count}}"
+                max="{{summaryInformation.quota.hard.count}}" data-displayval="100%"></progress>
+            </div>
+          </div>
+        </div>
+        <div class="display-flex quotas-progress">
+          <label class="mr-1">{{'SUMMARY.STORAGE_CONSUMPTION' | translate}}</label>
+          <label class="progress-label">
+              {{ summaryInformation.quota.hard.storage !== -1 ?(getIntegerAndUnit(summaryInformation.quota.hard.storage, summaryInformation.quota.used.storage).partNumberUsed
+                + getIntegerAndUnit(summaryInformation.quota.hard.storage, summaryInformation.quota.used.storage).partCharacterUsed) : getSuitableUnit(summaryInformation.quota.used.storage)}}
+
+            <!-- {{ getSuitableUnit(summaryInformation.quota.used.storage) }}  -->
+            {{ 'QUOTA.OF' | translate }}
+            {{ summaryInformation.quota.hard.storage ===-1? ('QUOTA.UNLIMITED' | translate) : getIntegerAndUnit(summaryInformation.quota.hard.storage, summaryInformation.quota.used.storage).partNumberHard }}
+             {{ summaryInformation.quota.hard.storage ===-1? '':  getIntegerAndUnit(summaryInformation.quota.hard.storage, summaryInformation.quota.used.storage).partCharacterHard }}
+          </label>
+        </div>
+
+        <div>
+          <div class="progress-block progress-min-width progress-div">
+            <div class="progress success"
+            [class.danger]="summaryInformation.quota.hard.storage!==-1?summaryInformation.quota.used.storage/summaryInformation.quota.hard.storage>0.9:false">
+              <progress
```

Comment:
what if `summaryInformation.quota.hard.storage` === 0

Reply:
There is no such possibility.